### PR TITLE
Disable routes to nodes at the unknown state

### PR DESCRIPTION
**Pull request by hackaugusto**
Users are experiencing problems with transfer because unknown nodes are actually unreachable. Issues #1086 and #1020

### CODE REVIEW

File: raiden/tests/unit/test_transfer.py [hunk 1]
```diff
@@ -356,124 +360,156 @@ def test_cancel_transfer(raiden_chain, token, deposit):
...
+    # path
+    identifier_refund = 3
+    amount_refund = 50
+    async_result = app0.raiden.mediated_transfer_async(
+        token,
+        amount_refund,
+        app2.raiden.address,
+        identifier_refund,
+    )
+    assert async_result.wait() is False, 'there is not path with capacity, the transfer must fail'
```
Comment (LefterisJP) [inline raiden/tests/unit/test_transfer.py]: `not path with capacity` -> `no path with enough capacity`

File: raiden/tests/utils/transport.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+class MessageLoggerTransport(DummyTransport):
+    def __init__(
+            self,
+            host,
+            port,
+            protocol=None,
+            throttle_policy=DummyPolicy()):
+
+        super(MessageLoggerTransport, self).__init__(host, port, protocol, throttle_policy)
+        self.addresses_to_messages = {}
```
Comment (LefterisJP) [inline raiden/tests/utils/transport.py]: can you initialize the dict with `dict()`? As I said above I thought we had agreed on this. It's not in the style guide though. If we are still in agreement about it I will add it.

File: raiden/network/channelgraph.py [hunk 3]
```diff
@@ -134,15 +133,21 @@ def get_best_routes(
     # distributable amount, but to sort them based on available balance and
     # let the task use as many as required to finish the transfer.
 
-    online_nodes = list()
-    unknown_nodes = list()
+    online_nodes = []
```
[the remark this answers is not in the document]
Comment (hackaugusto) [reply raiden/network/channelgraph.py]: okay, will change it back, we need to add this to the contributing style guide

[the remark this answers is not in the document]
Comment (LefterisJP) [reply raiden/network/channelgraph.py]: thanks, will do so myself in a PR now.

**Comment by LefterisJP** [comment]
looks good to me,.

**Pull request merged**